test/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/ProductDetailsViewSpec.scala

```diff
@@ -209,227 +209,253 @@ class ProductDetailsViewSpec extends ViewSpec {
     accessibilityStatementLink = Some("software-vendor-accessibility.com")
   )
 
-  "ProductDetailsPage" must {
-    "have a breadcrumb menu" which {
-      "contains the guidance page" in {
-        val link = document().selectNth(".govuk-breadcrumbs__list-item", 1).selectHead("a")
-        link.text shouldBe "Guidance"
-        link.attr("href") shouldBe appConfig.guidance
-      }
+  private val softwareVendorModelWithoutIncomeAndDeductions = softwareVendorModelFull.copy(incomeAndDeductions = Seq.empty)
+
+  private val softwareVendorModelWithoutFilters = softwareVendorModelFull.copy(filters = Seq.empty[VendorFilter])
+
+  private val softwareVendorModelWithoutAccessibilityLink = softwareVendorModelFull.copy(accessibilityStatementLink = None)
+
+  "ProductDetailsPage" when {
+    "the vendor has everything" when {
+      "the income and deductions feature switch is on" must {
```

Comment:
These tests need to be restructured because the only tests that need the feature switch to be on are `display the incomes and deductions section with all values` and `the vendor has minimal incomes and deductions supported`. The rest of the tests can remain outside of the `the income and deductions feature switch is on` block.